cmd/kyma/alpha/create/module/opts.go

```diff
@@ -8,16 +8,17 @@ import (
 type Options struct {
 	*cli.Options
 
-	ModPath        string
-	RegistryURL    string
-	Credentials    string
-	TemplateOutput string
-	Channel        string
-	Token          string
-	Insecure       bool
-	ResourcePaths  []string
-	Overwrite      bool
-	Clean          bool
+	ModPath           string
+	RegistryURL       string
+	Credentials       string
+	TemplateOutput    string
+	Channel           string
+	Token             string
+	Insecure          bool
+	ResourcePaths     []string
+	Overwrite         bool
+	Clean             bool
+	ValidateDefaultCR bool
```

Comment:
Do we really need this field? I can't think of a use case where we want to allow unvalidated CRs into the image.

Reply:
This validation takes around 5 seconds on average (depends on the machine). I think people will want to skip it in most cases during development, because waiting 5 seconds each time even if I don't change anything in CR/CRD is painful.
Maybe there's another way? We don't mind if the local image is not validated - after all it's on the local machine only. We don't want it to be published. Maybe the validation should be only required when publishing the module? The next [issue](https://github.com/kyma-project/lifecycle-manager/issues/226) will integrate pushing the module with the `kyma alpha create module` command. Perhaps we can solve this problem in that other issue.

Reply:
Good point yes! In that case I would say it makes sense to validate CR always when pushing the image.

Pushing is already implemented and working, see:
https://github.com/kyma-project/cli/blob/main/cmd/kyma/alpha/create/module/module.go#L132

Reply:
Done.